src/dialogs/quick-bar/ha-quick-bar.ts

```diff
@@ -72,22 +80,30 @@ export class QuickBar extends LitElement {
           )}
           type="search"
           value=${this._commandMode ? `>${this._itemFilter}` : this._itemFilter}
+          @keydown=${this._handleInputKeyDown}
+          @focus=${this._resetActivatedIndex}
         ></paper-input>
         ${this._commandMode
-          ? this.renderCommandsList(this._itemFilter)
-          : this.renderEntityList(this._itemFilter)}
+          ? this.renderCommandsList(this._itemFilter, this._activatedIndex)
+          : this.renderEntityList(this._itemFilter, this._activatedIndex)}
       </ha-dialog>
     `;
   }
 
-  protected renderCommandsList = memoizeOne((filter) => {
+  protected renderCommandsList = memoizeOne((filter, activatedIndex) => {
```

Comment:
The memoize makes a lot less sense now that we rerender it almost always anyway

Reply:
Yeah, though even in the first PR, the only case where we got to hit the cache is when you open the dialog, type nothing, close the dialog, then re-open the dialog. Filter string stays empty throughout so it would hit the cache.

Shall I remove it?

Reply:
Yeah

Reply:
Done